trino/sqlalchemy/dialect.py

```diff
@@ -84,6 +84,12 @@ def create_connect_args(self, url: URL) -> Tuple[Sequence[Any], Mapping[str, Any
             kwargs["http_scheme"] = "https"
             kwargs["auth"] = BasicAuthentication(url.username, url.password)
 
+        if "accessToken" in url.query:
+            kwargs["auth"] = JWTAuthentication(url.query["accessToken"])
+
+        if "http_scheme" in url.query:
```

Comment:
`http_scheme="https"` is required when used with any authentication.

Reply:
Yes, but I set schema to HTTPS when any authentication, not just JWT.
```diff
         if url.password:
             if not url.username:
                 raise ValueError("Username is required when specify password in connection URL")
-            kwargs["http_scheme"] = "https"
             kwargs["auth"] = BasicAuthentication(url.username, url.password)
 
+        if "accessToken" in url.query:
+            kwargs["auth"] = JWTAuthentication(url.query["accessToken"])
+
+        if "http_scheme" in url.query:
+            kwargs["http_scheme"] = url.query["http_scheme"]
+        if kwargs.get("auth"):
+            kwargs["http_scheme"] = "https"
```

Reply:
@dungdm93 can you please advise how to proceed after my latest changes

Reply:
LGTM. But seem you need to sign the CLA first.
@ebyhr could you please help him.